src/SWP/Bundle/TemplatesSystemBundle/Factory/ContainerFactory.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of the Superdesk Web Publisher Template Engine Bundle.
+ *
+ * Copyright 2015 Sourcefabric z.u. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2015 Sourcefabric z.ú
+ * @license http://www.superdesk.org/license
+ */
+
+namespace SWP\Bundle\TemplatesSystemBundle\Factory;
+
+use SWP\Component\Storage\Factory\FactoryInterface;
+
+class ContainerFactory
```

Comment:
default is `SWP\Component\Storage\Factory\Factory` which should be automatically registered. It does exactly the same as this class.